crates/tauri-plugin-holochain-service/android/src/main/java/com/plugin/holochain_service/HolochainServicePlugin.kt

```diff
@@ -87,8 +93,10 @@ class HolochainServicePlugin(private val activity: Activity): Plugin(activity) {
     fun installApp(invoke: Invoke) {
         Log.d(TAG, "installApp")
         val args = invoke.parseArgs(InstallAppPayloadFfiInvokeArg::class.java)
-        this.serviceClient.installApp(args.toFfi().toParcel())
-        invoke.resolve()
+        serviceScope.launch(Dispatchers.IO) {
```

Comment:
Client functions are now `suspend` (async), so we need to call them inside a `Coroutine` (thread block). Make the Coroutine run on the `Dispatchers.IO` thread, to avoid blocking the main thread.